src/NuGet.Core/NuGet.Commands/RestoreCommand/RestoreResult.cs

```diff
@@ -171,7 +179,9 @@ public ISet<LibraryRange> GetAllUnresolved()
                 }
                 else
                 {
-                    log.LogMinimal($"Writing lock file to disk. Path: {result.LockFilePath}");
+                    log.LogMinimal(string.Format(CultureInfo.CurrentCulture,
```

Comment:
I don't think any callers need this on Minimal anymore (there might be a test that verifies it is logged there). I suggest moving this to Information. Restore writes way too much into Minimal already

Same for tools